### PR TITLE
Javascript Loops

### PR DESCRIPTION
**Pull request by louiewhitz**
Screen Shot 2022-05-19 at 2 26 46 PM
Screen Shot 2022-05-19 at 2 26 38 PM

### CODE REVIEW

File: javascript-loops/main.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+var numbers = [];
+var currentNumbers = 1;
```
Comment (codingcodymiller) [inline javascript-loops/main.js]: You should not be making variables for use in a function outside of the function definition, that means these variables will continue to exist longer than they need to, which is not ideal. If you only intend to use a variable inside of a single function, that variable should be declared inside the function. Make sure you address this in your other function definitions as well.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-loops/main.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  while (currentNumbers <= 11) {
+    numbers.push(currentNumbers);
+    currentNumbers++;
+  }
+  return numbers;
+}
+console.log(getNumbersToTen());
+var evenNumbers = [];
+var currentNumber = 2;
```
Comment (codingcodymiller) [inline javascript-loops/main.js]: My comment in my previous review applies here.

File: javascript-loops/main.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+function doubleAll(numbers) {
+  var doubled = [];
+  for (var i = 0; i < numbers.length; i++) {
+    doubled.push((numbers[i]) * 2);
+  }
+  return doubled;
+}
+console.log(doubleAll([2, 5, 7, 8]));
+var keys = [];
```
Comment (codingcodymiller) [inline javascript-loops/main.js]: My comment in my previous review applies here.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by louiewhitz** [comment]
Screen Shot 2022-05-27 at 7 05 33 PM
Screen Shot 2022-05-27 at 7 05 24 PM

File: javascript-loops/main.js (hunk 3 above)
Comment (codingcodymiller) [reply javascript-loops/main.js]: This `keys` variable still needs to be moved inside the function block.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by louiewhitz** [comment]
Screen Shot 2022-06-
![Screen Shot 2022-06-09 at 11 09 26 PM
09 at 11 16 36 PM](https://user-images.githubusercontent.com/99847558/173003219-2b81a357-6e32-4828-a1f7-2fb6ee53b19d.png)

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**